dandi/tests/test_register.py

```diff
@@ -1,14 +1,13 @@
 import re
 
-import yaml
+import ruamel.yaml
 
 from ..consts import dandiset_identifier_regex, dandiset_metadata_file
 from ..register import register
 
 
 def yaml_load(s):
```

Comment:
This seems to be a common pattern.  May be we could just move this helper into `dandi.utils` to reside alongside with `yaml_dump`, and just equip with `typ=None` kwarg so where needed we could switch to `safe` or any other?

Reply:
Done.